.circleci/config.yml

```diff
@@ -4,110 +4,110 @@
 #
 version: 2
 references:
-  container_python38: &container_python38
+  container_python310: &container_python310
     docker:
-      - image: circleci/python:3.8.1
+      - image: cimg/python:3.10.11
       - name: localstack
-        image: localstack/localstack
+        image: localstack/localstack:2.0.2
     working_directory: ~/repo
-
-  container_python36: &container_python36
+  
+  container_python38: &container_python38
     docker:
-      - image: circleci/python:3.6
+      - image: circleci/python:3.8.1
```

Comment:
Not super concerned about this since it follows the convention we have but why is the circleci image python version not being upgraded to 3.10.11?

Reply:
this is a transition to circle-ci's suggested "next gen convencience images" as documented https://circleci.com/docs/next-gen-migration-guide/
here

Reply:
sorry, actually a correction cause I think I misread:
3.10 is updated to that next gen image because it's "new" I've left 3.8 at it's current configuration because the paradigm is making sure the "current system still works"